### DISCUSSION TITLE
Branch merge is not working

### DISCUSSION THREAD

**Discussion by examsOfBharat** [community/community#130256] [Pull Requests]
### Select Topic Area

Product Feedback

### GitHub Feature Area

Pull Requests

### Body

There is issue with pull request, we are not able to merge branch, it has stuck, tried multiple times but gives same result

Checking for ability to merge automatically…
Hang in there while we check the branch’s status.

this is showing from last 20 mins

**Comment by AsmaeelAli** [comment]
I have the same problem on private repository.

**Comment by Moonwilles** [comment]
Despite reports that the problem has been fixed, my workflow never started and remained stuck on 'waiting for status to be reported'.

**Reply by XihuaYang** [reply]
Our alternate solution when the issue is still there is to perform the merge from command line, this bypass the Github page but will have a normal Git history, subsequent workflows (Jenkins, actions etc.) can be triggered normally

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @examsOfBharat ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Comment by rlzyy** [comment]
Omg
